data/mock/cacherMock.go

```diff
@@ -103,7 +103,12 @@ func (cm *CacherMock) MaxSize() int {
 }
 
 // RegisterHandler -
-func (cm *CacherMock) RegisterHandler(func(key []byte, value interface{})) {
+func (cm *CacherMock) RegisterHandler(func(key []byte, value interface{}), string) {
```

Comment:
_ ..., _ ...

Reply:
actually, in this case it is better to leave it as it is (it is not a weak warning) and that function signature carries useful information (like the first parameter is the key and the last is the value).